guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -174,32 +245,72 @@ Incremental upgrade (of a content view):: The act of creating a new (minor) cont
+An installation medium in {Project} represents the installation files for one or more operating systems, which must be accessible over the network, either through an URL or an NFS server location.
+It is usually either a mirror or a CD or DVD image.
+// would you want to do this for RHEL or rather use a synchronized Kickstart repository?
+Pointing the URL of the installation medium to a local copy, for example `http://{foreman-example-com}/pub/installation_media/`, can improve provisioning time and reduce network load.
++
+Every operating system depends on exactly one path of an installation medium, whereas installation media paths may serve different operating systems at the same time.
+ifdef::satellite[]
+You
+endif::[]
+ifndef::satellite[]
+For {EL}, you
+endif::[]
+can use `$version`, `$major`, and `$minor` to parameterize the URL.
+ifndef::satellite[]
+Debian, Ubuntu, and {SLES} do not support such variables.
++
+You can bundle installation media for multiple versions of Debian or Ubuntu in one path, for example a repository containing packages for both _Debian 11 "Bullseye"_ and _Debian 12 "Bookworm"_.
+endif::[]
+
 [[Job]]
 Job:: A command executed remotely on a host from {ProjectServer}.
 Every job is defined in a job template.
 
 [[Katello]]
-Katello:: A Foreman plugin responsible for subscription and repository management.
+Katello::
+An open source Foreman plugin to perform lifecycle management of registered hosts, content management, and subscription handling.
+It depends on Pulp for content management, which fetches software from repositories and stores various versions of it.
+ifdef::satellite,orcharhino[]
+The Katello plugin is installed by default on {Project}.
+endif::[]
+
+[[Katello_Host_Tools_and_Katello_Host_Tools_Tracer]]
+Katello Host Tools and Katello Host Tools Tracer::
+The client-side software is responsible for the connection between hosts and content from {ProjectServer} or {SmartProxyServers}.
+It uploads currently used repositories and installed packages to {Project} and signals if the host needs to be rebooted after updating the Linux Kernel.
 
 [[Lazy_sync]]
 Lazy sync:: The ability to change the default download policy of a repository from *Immediate* to *On Demand*.
 The *On Demand* setting saves storage space and synchronization time by only downloading the packages when requested by a host.
 
 [[Location]]
-Location:: A collection of default settings that represent a physical place.
+Location::
+A collection of default settings that represent a physical place.
+Location is a tag mostly used for geographical separation of hosts within {Project}.
+Examples include different cities or different data centres.
 
 [[Library]]
 Library:: A container for content from all synchronized repositories on {ProjectServer}.
 Libraries exist by default for each organization as the root of every lifecycle environment path and the source of content for every content view.
 
 [[Lifecycle_environment]]
```

Comment:
I don't think facts like this should be documented in a glossary. There are fully fledged modules describing lifecycle environments.

Reply:
I’m going to make things complicated again (sorry!): We're thinking about a situation when a user reads section X, hits an unknown term ("lifecycle environment"), visits the glossary to look up a definition, and then clicks the link to section Y that explains LE.

Well, why not just link to section Y from section X?

Or is there a different content flow/user journey where a glossary is an indispensable link in the chain?

Reply:
Unless we find a better wording for "A typical scenario", then I'd actually leave it as is. Suggestions welcome; but unless you voice strong concerns, I consider these comments non-blocking.

I'd rather keep the glossary entry short because we have great docs on how to work with LE/LEPs.